recipes/phyfleaux/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020 R. Tohid
+#
+# Distributed under the Boost Software License, Version 1.0. (See a copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+FROM ubuntu:18.04
+
+RUN apt update
+RUN apt install -y sudo
+
+# add user
+RUN adduser --disabled-password --gecos '' stellar
+RUN adduser stellar sudo
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# OS Packages
+RUN apt install -y python3.8 python3.8-dev python3.8-distutils
+RUN apt install -y build-essential
+RUN apt install -y cmake automake autoconf libtool lld
+RUN apt install -y libpng-dev libjpeg-dev
+RUN apt install -y zlib1g-dev libz-dev
+RUN apt install -y git vim
+RUN apt install -y wget curl
+RUN apt install -y bash-completion
+RUN apt install -y llvm-8 llvm-8-dev clang-8
+
+# install pip3.8
```

Comment:
3.8 should probably be an environment variable, argument, or parameter. Same for the -8 on llvm.

Reply:
You're right, I will parametrize all as soon as I get the chance. Meanwhile, I just want to make sure everybody has a working environment for papers they're working on.

Reply:
BTW, the image would be smaller if you compressed all this into a single apt-get call.